session/session.go

```diff
@@ -825,12 +825,92 @@ func (s *session) SetGlobalSysVar(name, value string) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+
+	switch name {
+	case variable.TiDBMaxChunkSize:
+		return s.persistVarInConstraint(name, sVal, variable.TiDBInitChunkSize, func(s *session, val string) (string, error) {
+			newMaxChunkSize, err := strconv.Atoi(sVal)
+			if err != nil {
+				return "", errors.Trace(err)
+			}
+			initChunkSize, err := strconv.Atoi(val)
+			if err != nil {
+				return "", errors.Trace(err)
+			}
+			if initChunkSize > newMaxChunkSize {
+				return "", errors.Errorf("Try to set TiDB MaxChunkSize to %d small than InitChunkSize %d, please change InitChunkSize ahead",
+					newMaxChunkSize, initChunkSize)
+			}
+			return strconv.Itoa(newMaxChunkSize), nil
+		})
+	case variable.TiDBInitChunkSize:
+		return s.persistVarInConstraint(name, sVal, variable.TiDBMaxChunkSize, func(s *session, val string) (string, error) {
+			newInitChunkSize, err := strconv.Atoi(sVal)
+			if err != nil {
+				return "", errors.Trace(err)
+			}
+			maxChunkSize, err := strconv.Atoi(val)
+			if err != nil {
+				return "", errors.Trace(err)
+			}
+			if newInitChunkSize > maxChunkSize {
+				return "", errors.Errorf("Try to set TiDB InitChunkSize to %d big than MaxChunkSize %d, please change MaxChunkSize ahead",
```

Comment:
how about "tidb-init-chunk-size(%d) cannot be larger than tidb-max-chunk-size(%d), please increase tidb-max-chunk-size first"?